JS/index.scss

```diff
@@ -0,0 +1,217 @@
+$body-txt:"100px";
+body{
+	margin:0px 0px 0px 0px;
+}
+header{
+	width:100%;
+	div{
+		height: 55px;
+		width: 100%;
+		background-color: rgba(33, 33, 29, 0.92);
+		//margin-top: 0px;
+		ul{
+			display: flex;
+			//justify-content: space-between;
+			width: 100%;
+			margin-left: 15px;
+			margin-right: 30px;
+			margin-top:0px;
+			margin-bottom: 0px;
+			height: 100%;
+			list-style-type: none;
+			li{
+				//text-decoration: none;
+				//background-color:black;
+				list-style-type: none;
+
+				a{
+					text-decoration:none;
+					color: white;
+					font-size: 13px;
+					margin-top: 5px;
+				}
+				input{
+					height:100%;
+					width:100%;
+					background-color:#32322F;
+					color:white;
+					border:none;
+				}
+			}
+		}
+	}
+}
+ul.leftListHeader{
+	
+	justify-content: flex-start;
+	height: 100%;
+	li.header-text{
+		
+		padding-top: 20px;
+		//margin-top: 6px;
+		border-right:1px solid red;
+		width:90px;
+		text-align: center;
+	}
+	li.logo{
+		width: 45px;
+		padding-left:15px;
+		border-right: 1px solid red;
+		border-left:1px solid red; 
+		background-color:72, 64, 64, 0.92;
+		padding-top:15px;
+	}
+	
+}
+ul.rightListHeader{
+	justify-content: flex-end;
+	margin-right: 180px;
+	margin-left: 100px;
+	li{
+		//padding-top: 20px;
+		border-right:1px solid red;
+		
+	}
+}
+li.left-text-header{
+	border-left:1px solid red;
+	width: 75px;
+	text-align: center;
+	padding-top:20px;
+}
+li.right-text-header{
+	width:130px;
+	text-align: center;
+	margin-top:5px;
+}
+footer{
+	width:100%;
+	div{
+		height: 65px;
+		background-color:gray;
+		ul{
+			display: flex;
+			//justify-content: space-between;
+			padding-top:10px;
+			list-style-type: none;
+			width:100%;
+			li{	
+				a{
+					text-decoration: none;
+					color: white;
+					font-size: 11px;
+				}
+			}
+		}
+	}
+}
+ul.leftListFooter{
```

Comment:
don't prepend the tag name to classes
You will have to change it for all the css that u have written